week-05/04_pig_latin/pig_latin.rb

```diff
@@ -1 +1,38 @@
-#write your code here
+def translate(prompt)
```

Comment:
Looks like spacing/indentation is little off that may need some fixing for translate method